src/main/java/io/strimzi/kafka/metrics/http/HttpServers.java

```diff
@@ -53,22 +53,33 @@ public synchronized static void release(ServerCounter serverCounter) {
      * Class used to keep track of the HTTP server started on a listener.
      */
     public static class ServerCounter {
+        private HTTPServer.Builder builder;
+        private Listener listener;
+        private AtomicInteger count;
```

Comment:
Why are we making these fields not `final`?

Reply:
Fixed.